Formula/d/docker-buildx.rb

```diff
@@ -1,8 +1,8 @@
 class DockerBuildx < Formula
   desc "Docker CLI plugin for extended build capabilities with BuildKit"
   homepage "https://docs.docker.com/buildx/working-with-buildx/"
-  url "https://github.com/docker/buildx/archive/refs/tags/v0.16.0.tar.gz"
-  sha256 "6660afc93ed7b6f1046cf179b780d8c0d9ee35fc3ac8ace78af49b7ea63969fc"
+  url "https://github.com/docker/buildx/archive/refs/tags/v0.16.1.tar.gz"
```

Comment:
Not yet as latest release on https://github.com/docker/buildx/releases
<img width="818" alt="image" src="https://github.com/user-attachments/assets/ae72084a-4127-4d2f-880f-9f4ba686ad1b">